keanu-project/src/test/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/operators/binary/ConcatenationVertexTest.java

```diff
@@ -225,10 +244,15 @@ public void canConcatenateSimpleAutoDiffForwardSharedParentsAndDifferentSize() {
         ConcatenationVertex concat = new ConcatenationVertex(1, e, f);
```

Comment:
Weird that we'll still doing this, I must've forgot to add it to the vertex. Can you be a hero and add a `concat` to `DoubleVertex`?

Reply:
As discussed - I'll do this as part of the change to the `concat` method itself